packages/form/src/widgets/TextareaWidget/TextareaWidget.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { TextField, Icon } from '@equinor/eds-core-react'
+
+import { error_filled } from '@equinor/eds-icons'
+
+Icon.add({ error_filled })
+
+const TextareaWidget = (props: any) => {
+  const { label, onChange } = props
+
+  const _onChange = ({
+    target: { value },
+  }: React.ChangeEvent<HTMLInputElement>) => onChange(value === '' ? '' : value)
+  return (
+    <TextField
+      {...props}
+      multiline={true}
+      rows={5}
+      onChange={_onChange}
```

Comment:
is onChange prop to TextFiled defined two times here?
since onChange is part of {...props} and added on line 20

Reply:
I think that it only will be added once since the one on line 20 will override the first one.